.lintrunner.toml

```diff
@@ -0,0 +1,144 @@
+# Configuration for lintrunner https://github.com/suo/lintrunner
+
+[[linter]]
+code = 'FLAKE8'
+include_patterns = [
+    '**/*.py',
+    '**/*.pyi',
+]
+exclude_patterns = [
+    '.git/**',
+    '.lintrunner/**',
+    'docs/**',
+]
+command = [
+    'python3',
+    '.lintrunner/flake8_linter.py',
+    '--',
+    '@{{PATHSFILE}}'
+]
+init_command = [
+    'python3',
+    '.lintrunner/pip_init.py',
+    '--dry-run={{DRYRUN}}',
+    'flake8==5.0.4',
+    'flake8-bugbear==22.10.27',
+    'flake8-pyi==22.10.0',
+    'dlint==0.13.0',
+    'toml==0.10.2',  # Needed for pydocstrings to parse pyproject.toml
+    'flake8-docstrings==1.6.0',
+]
+
+# [[linter]]
+# code = 'MYPY'
```

Comment:
Is MYPY still ignored even though it has linter.py?

Reply:
Yes. We need to gradually enable mypy. Likely in subsequent prs.